src/web/mjs/connectors/templates/MH.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class MH extends Connector {
+
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = [];
+        this.url = undefined;
+
+        this.path = '/booklist?page=';
```

Comment:
May use a placeholder to allow better control for websites that do not use the page
number as last element, e.g.
`'/booklist?page=%PAGE%';`